clusterloader2/testing/density/config.yaml

```diff
@@ -20,6 +20,7 @@ steps:
     Method: APICallLatency
     Params:
       action: reset
+# Creating saturation pods
```

Comment:
Can you use simple present tense (i.e "Create") to be consistent with your load test comments?

Reply:
Done.